protostar/argument_parser/argument.py

```diff
@@ -11,7 +12,9 @@ class Argument(Generic[ArgTypeNameT]):
     type: ArgTypeNameT
     is_positional: bool = False
     is_required: bool = False
-    is_array: bool = False
+    multiple_args_resolver: Literal[
+        "single_element", "list", "list_or_dict"
+    ] = "single_element"
```

Comment:
I thought about this name and I think I much more prefer [`clap`'s `value_parser` approach ](https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes). maybe we should adopt something like this here?

at least use `value_parser` name instead of `multiple_args_resolver`

Reply:
[rename multiple_args_resolver => value_parser](https://github.com/software-mansion/protostar/pull/1140/commits/8d118878c224c07bec6b22f6405259e8f6ab8a57)